packages/teams-js/src/public/dialog.ts

```diff
@@ -88,10 +88,13 @@ export namespace dialog {
       registerHandler('messageForParent', messageFromChildHandler);
     }
     const dialogInfo: DialogInfo = getDialogInfoFromUrlDialogInfo(urlDialogInfo);
-    sendMessageToParent('tasks.startTask', [dialogInfo], (err: string, result: string | object) => {
-      submitHandler({ err, result });
-      removeHandler('messageForParent');
-    });
+    const submitHandlerHelper = submitHandler
+      ? (err: string, result: string | object) => {
+          submitHandler({ err, result });
+          removeHandler('messageForParent');
+        }
+      : null;
```

Comment:
Whenever I see `null` being used I really want to know if there's a reason for it to be `null` and not `undefined`.